app/component-library/components/Sheet/SheetBottom/SheetBottom.tsx

```diff
@@ -57,6 +57,13 @@ import {
   SheetBottomRef,
 } from './SheetBottom.types';
 
+/**
+ * @deprecated The `<SheetBottom>` component has been deprecated in favor of the new `<BottomSheet>` component from the component-library.
+ * Please update your code to use the new `<BottomSheet>` component instead, which can be found at app/component-library/components/BottomSheets/BottomSheet/BottomSheet.tsx.
+ * You can find documentation for the new BottomSheet component in the README:
+ * {@link https://github.com/MetaMask/metamask-mobile/tree/main/app/component-library/components/BottomSheets/BottomSheet/README.md}
+ * If you would like to help with the replacement of the old SheetBottom component, please create a new GitHub issue to address a Pull request to it.
```

Comment:
I've created a good first issue that contributors can reference if they'd like to replace the old SheetBottom with BottomSheet

```suggestion
 * If you would like to help with the replacement of the old `OLD_COMPONENT` component, please submit a pull request against this GitHub issue:
 * {@link https://github.com/MetaMask/metamask-mobile/issues/7658}
```

Reply:
Thank you so much for that @georgewrmarshall <3